peachjam/adapters/adapters.py

```diff
@@ -319,6 +320,15 @@ def set_parent(self, imported_document, created_document):
             created_document.parent_work = None
         created_document.save()
 
+    def remove_commencement_relationships(self, created_document):
```

Comment:
I suggest doing this as a part of `fetch_relationships`.

Also `repealed-by` and `amended-by` must also be cleaned up.

Reply:
Ok, changed :+1: